src/pcre2_compile.c

```diff
@@ -7102,7 +7102,6 @@ for (;; pptr++)
     single-char opcodes. */
 
     reqvary = (repeat_min == repeat_max)? 0 : REQ_VARY;
-    op_type = 0;
```

Comment:
Clang is correct here. The assignment to `op_type` is provably a dead assignment. What's more... I reckon it hurts rather than helps, because if there's a branch in the code that _should_ be assigning to op_type but forgets, this would suppress warnings about use-of-uninitialised!